requirements/analyzer.txt

```diff
@@ -42,7 +42,7 @@ cymem==2.0.8
     #   preshed
     #   spacy
     #   thinc
-cytoolz==0.12.3
+cytoolz==1.0.0
```

Comment:
This is probably worth a test to make sure this works before merging.

Reply:
Looks pretty minor, probably just got sick of not being 1.0 :) 

https://github.com/pytoolz/cytoolz/releases/tag/1.0.0